### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by dogan2893** [Chia-Network/chia-blockchain#5003] [Support]
Only wrote 3072 of 1048564 bytes at offset 268432384 to "C:\\chia\\plot-k32-2021-05-14-12-47-b0cdf7eb7963ee73e3aa9a58d553a08ed2d907041b08cca59896a7b84b725308.plot.p1.t5.sort_bucket_072.tmp"with length 268435456. Error 1. Retrying in five minutes.
Only wrote 3072 of 1048564 bytes at offset 268432384 to "C:\\chia\\plot-k32-2021-05-14-12-47-b0cdf7eb7963ee73e3aa9a58d553a08ed2d907041b08cca59896a7b84b725308.plot.p1.t5.sort_bucket_007.tmp"with length 268435456. Error 1. Retrying in five minutes.

Any body help about this error ?

**Comment by LewyMaster** [comment]
Yes. its simple. your disk space is probably full. Dont worry. it can be resolved without deleting unfinished plot.

just try this:

1. Open your temp directory (in your case "C:\chia\" ? )
2. Find biggest *.plot.tableX.tmp file with the same ID in name (for you it could be "plot-k32-2021-05-14-12-47-b0cdf7eb7963ee73e3aa9a58d553a08ed2d907041b08cca59896a7b84b725308.plot.table2.tmp)
WARNING! It must be file which should NOT be currently processed! (safest way is use table with 2 numbers bigger or smaller from your currently processing table - eg. if you computing/Backpropagating on table 4 then check table 1/2 or 6/7 )
3. This files should have weight about 40-50GiB. try to move this files to other disk (try one per time) then....
4. ...create symlink to this file in the directory you cut file from (YES you can create symlinks in Windows OS). 
 
Example:
`mklink C:\chia\plot-k32-2021-05-14-12-47-b0cdf7eb7963ee73e3aa9a58d553a08ed2d907041b08cca59896a7b84b725308.plot.table2.tmp D:\plot-k32-2021-05-14-12-47-b0cdf7eb7963ee73e3aa9a58d553a08ed2d907041b08cca59896a7b84b725308.plot.table2.tmp`

in general: 
`mklink <where_you_need_symlink_to_be> <to_which_file_it_link>`

Please be careful and do point 4. AFTER moving your .tmp file.

if you do everything as instructed then you will free up disk space to continue plotting but also not break the process because by creating .symlinks you provide chia app how to find moved files.


Little suggestion:
If you trying to plot on your system disk ("C:\") then it is very exposed to damage and data loss. Try to find disk dedicated for plotting (even if it not fast).

Let me know if it works for you..
(and Im sorry for my English :) )

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
